gradoop-common/src/main/java/org/gradoop/common/storage/api/EPGMStore.java

```diff
@@ -28,18 +28,22 @@
  * The EPGM store is responsible for writing and reading graph heads, vertices
  * and edges.
  *
+ * @param <C> gradoop flink config
  * @param <G> EPGM graph head type
  * @param <V> EPGM vertex type
  * @param <E> EPGM edge type
  */
-public interface EPGMStore
-  <G extends EPGMGraphHead, V extends EPGMVertex, E extends EPGMEdge> {
+public interface EPGMStore<
+  C extends GradoopConfig,
```

Comment:
should be GradoopStoreConfig since it is a generic parameter to EPGMStore interface

Reply:
already changed